share/picongpu/examples/atomicPhysics/README.rst

```diff
@@ -0,0 +1,33 @@
+atomicPhysics: atomic Physics example for PIConGPU based on the Empty example by Axel Huebl
+============================
+
+.. sectionauthor:: Axel Huebl <a.huebl (at) hzdr.de> and Brian Marre <b.marre (at) hzdr.de>
```

Comment:
If this is your test case, you might remove @ax3l here.

@ax3l Do you agree with that?